spaceship/lib/spaceship/test_flight/base.rb

```diff
@@ -4,6 +4,20 @@ def self.client
       @client ||= Client.client_with_authorization_from(Spaceship::Tunes.client)
     end
 
+    ##
+    # Have subclasses inherit the client from their superclass
+    #
+    # Essentially, we are making a class-inheritable-accessor as described here:
+    # https://apidock.com/rails/v4.2.7/Class/class_attribute
+    #
+    # TODO[snatchev]: move this into spaceship/base.rb once all the tests pass
```

Comment:
Sorry, that was a cryptic comment - are we now in a place where this TODO could be resolved?